### ISSUE TITLE
import more standard names for genes

### ISSUE THREAD

**Issue by vincerubinetti** [greenelab/py3-adage-backend#37]
In a meeting with Deb and Georgia, it was said that some of the genes in our database do have known `standard_name`s, but show up as a duplicate of the `systematic_name` instead.

For example, the gene with the systematic name `PA0690` also has the standard name `pdtA`, but our database shows `PA0690` for both the systematic name and the standard name.

We should look into pulling more up to date information into the database.

**Comment by cgreene** [comment]
looks like pdtA instead of anr - http://www.pseudomonas.com/feature/show/?locus_tag=PA0690 - but agree this was the point 👍

**Comment by dongbohu** [comment]
The management command that we used to populate the gene tables is pretty old. It is probably a good time to rewrite it in Python 3.

**Comment by dongbohu** [comment]
`pdtA` is not included in the gene_info file:
`ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Archaea_Bacteria/Pseudomonas_aeruginosa_PAO1.gene_info.gz`
nor is it in gene_history file: `ftp://ftp.ncbi.nih.gov/gene/DATA/gene_history.gz`

So I thought I could extract it from the file that I used to get the PA14 names:
http://pseudomonas.com/downloads/pseudomonas/pgd_r_19_1/Pseudomonas_aeruginosa_PAO1_107/Pseudomonas_aeruginosa_PAO1_107_orthologs.csv.gz
The following two lines are copied from this file (the first line is the header)
```
"Strain(Query)","Locus Tag(Query)","Description(Query)","Strain(Hit)","Locus Tag(Hit)","Description(Hit)","Percent Identity","Alignment Length"
"Pseudomonas aeruginosa PAO1 (Reference)","PA0690","phosphate depletion regulated TPS partner A, PdtA","Pseudomonas aeruginosa UCBPP-PA14","PA14_55400","hypothetical protein","59.0","4264"
```
To extract `pdtA`, I thought I could read the `Description(Query)` column (column 3) and treat the first alphanumeric substring right after `, ` as a gene's valid standard name, but that is not true for some genes such as `PA2683`, whose `Description(Query)` column is:
```
probable serine/threonine dehydratase, degradative
```
Here `degradative` is NOT the standard name of `PA2683`. 

Attached is a list of PAO1 name (column 2) and `Description(Query)` (column 3) extracted from the online file:
[col_2_3.txt](https://github.com/greenelab/py3-adage-backend/files/4494424/col_2_3.txt)
You can see that `PA2683` is not the only exception. Here are the obvious exceptions in this file:
```
PA0024 	 coproporphyrinogen III oxidase, aerobic
PA0331 	 threonine dehydratase, biosynthetic
PA0347 	 glycerophosphoryl diester phosphodiesterase, periplasmic
PA1326 	 threonine dehydratase, biosynthetic
PA1614 	 glycerol-3-phosphate dehydrogenase, biosynthetic
PA5172 	 ornithine carbamoyltransferase, catabolic
PA2683 	 probable serine/threonine dehydratase, degradative
PA3537 	 ornithine carbamoyltransferase, anabolic
```

Is there another file online that is more suitable for this purpose?

**Comment by dongbohu** [comment]
Closed by PR #50.
